packages/riverpod_lint_flutter_test/test/assists/convert_to_consumer_widget_test.dart

```diff
@@ -3,441 +3,204 @@ import 'dart:io';
 import 'package:analyzer/dart/analysis/results.dart';
 import 'package:analyzer/dart/analysis/utilities.dart';
 import 'package:analyzer/source/source_range.dart';
+import 'package:pubspec_parse/pubspec_parse.dart';
 import 'package:riverpod_lint/src/assists/convert_to_stateful_base_widget.dart';
 import 'package:riverpod_lint/src/assists/convert_to_stateless_base_widget.dart';
 import 'package:riverpod_lint/src/assists/convert_to_widget_utils.dart';
+import 'package:riverpod_lint/src/riverpod_custom_lint.dart';
 import 'package:test/test.dart';
 
 import '../golden.dart';
 
 void main() {
-  testGolden(
-    'Convert widgets to consumerwidgets',
-    'assists/convert_to_consumer_widget.json',
-    () async {
-      final assist = ConvertToStatelessBaseWidget(
-        targetWidget: StatelessBaseWidgetType.consumerWidget,
-      );
-      final file = File(
-        'test/assists/convert_to_consumer_widget.dart',
-      ).absolute;
-
-      final result = await resolveFile2(path: file.path);
-      result as ResolvedUnitResult;
-
-      var changes = [
-        // Stateless
-        ...await assist.testRun(result, const SourceRange(163, 0)),
-        ...await assist.testRun(result, const SourceRange(174, 0)),
-        ...await assist.testRun(result, const SourceRange(185, 0)),
-
-        // StatelessWithComma
-        ...await assist.testRun(result, const SourceRange(350, 0)),
-
-        // Hook
-        ...await assist.testRun(result, const SourceRange(524, 0)),
-
-        // HookConsumer
-        ...await assist.testRun(result, const SourceRange(690, 0)),
-
-        // Stateful
-        ...await assist.testRun(result, const SourceRange(884, 0)),
-
-        // ExplicitCreateState
-        ...await assist.testRun(result, const SourceRange(1208, 0)),
-
-        // HookStateful
-        ...await assist.testRun(result, const SourceRange(1553, 0)),
-
-        // ConsumerStateful
-        ...await assist.testRun(result, const SourceRange(1863, 0)),
-
-        // HookConsumerStateful
-        ...await assist.testRun(result, const SourceRange(2214, 0)),
-
-        // ConsumerWidget
-        ...await assist.testRun(result, const SourceRange(2582, 0)),
-      ];
-
-      expect(changes, hasLength(9));
-
-      return changes;
-    },
+  final pubspec = Pubspec(
```

Comment:
Could you have a test variant for if hooks_riverpod isn't here?

Reply:
I was testing pubspec as null, but I felt it was confusing, so I fixed it.
[a224e24](https://github.com/rrousselGit/riverpod/pull/2306/commits/a224e248342a8dc259da1ccfe64d366ef0bb0740)

Reply:
Much better, thanks!